app/src/parser_impl.c

```diff
@@ -1163,7 +1182,17 @@ parser_error_t _read(parser_context_t *c, parser_tx_t *v)
         CHECK_ERROR(_readTxAssetConfig(c, v))
         break;
     case TX_APPLICATION:
+        #if defined(TARGET_NANOS) || defined(TARGET_NANOS2) || defined(TARGET_NANOX) || defined(TARGET_STAX) || defined(TARGET_FLEX)
+        if (!app_mode_blindsign() && !app_mode_expert()) {
```

Comment:
I think I might be missing the goal here. So if Bs not enabled and app in normal mode, we request BS mode ?
So I guess the user cannot sign any tx with app in normal mode.

I was not really in the discussion of this but in my opinion, in this case the BS mode is not required because we will not show certain information, but more of a shortcut mode imitation, thus the user can enable it to skip screens.
I think here we are removing the possibility of the user to check all the screens if he wants.

Reply:
Just saw the 4 possible options on the Slack channel, I guess this is right then.